exit-from-dialog.js

```diff
@@ -0,0 +1,165 @@
+/*
+  ====================================
+  DIALOG EXIT FUNCTION MOD (@mildmojo)
+  ====================================
+
+  Lets you exit to another room from dialog (including inside conditionals). Use
+  it to make an invisible sprite that acts as a conditional exit, use it to warp
+  somewhere after a conversation, use it to put a guard at your gate who only
+  lets you in once you're disguised, use it to require payment before the
+  ferryman will take you across the river.
+
+  Using the {exitAfter} function in any part of a series of dialog will make the
+  game exit to the new room after the dialog is finished. Using {exitNow} will
+  cut off the dialog text and immediately warp to the new room.
+
+  Usage: {exitAfter "<room name>,<x>,<y>"}
+         {exitAfter "<room name>,<x>,<y>,sprite"}
+         {exitNow "<room name>,<x>,<y>"}
+         {exitNow "<room name>,<x>,<y>,sprite"}
+
+  Example: {exitAfter "FinalRoom,8,4"}
+           {exitNow "FinalRoom,8,11,sprite"}
+
+  WARNING: In exit coordinates, the TOP left tile is (0,0). In sprite coordinates,
```

Comment:
should move this above the usage+examples, seeing "sprite" in those is confusing until you've seen this bit